src/utils/api.js

```diff
@@ -4,8 +4,10 @@ const ncNewsApi = axios.create({
   baseURL: 'https://nc-news-gyvj.onrender.com/api/'
 })
 
-export const getArticles =  () => {
-  return ncNewsApi.get('articles').then((res) => res.data)
+export const getArticles =  (topic) => {
+  if (topic === "All") topic = "";
+  const query = topic ? `?topic=${topic}` : "";
+  return ncNewsApi.get(`articles${query}`).then((res) => res.data)
```

Comment:
This is a great way to handle the queries. Axios also has it's own way to take queries. get method can take an object as it's second argument `ncNewsApi.get('articles',{params:{topic})`